src/views/Home.vue

```diff
@@ -0,0 +1,10 @@
+<script setup lang="ts">
+// This starter template is using Vue 3 <script setup> SFCs
+// Check out https://v3.vuejs.org/api/sfc-script-setup.html#sfc-script-setup
+import About from "../components/About.vue";
+</script>
+
+<template>
+  <a href="/login">Login</a> |
+  <a href="/about">About</a>
```

Comment:
You should use router-links instead of a tags.

https://router.vuejs.org/guide/#router-link

This allows us to prevent reloading while navigating on our website.

Reply:
Checking the other branch, it seems like this isn't an issue there....